playground/TestPlatform.Playground/Program.cs

```diff
@@ -62,6 +63,38 @@ static void Main(string[] args)
         r.RunTestsWithCustomTestHost(sources, sourceSettings, options, new TestRunHandler(), new DebuggerTestHostLauncher());
     }
 
+    public class PlaygroundTestDiscoveryHandler : ITestDiscoveryEventsHandler, ITestDiscoveryEventsHandler2
```

Comment:
Kept it so that we don't have to create it again in other playground runs.

Reply:
Not necessary in this PR but we should make the handler do something similar as the other handler does, to make it more useful.

Reply:
Done.